Assets/Scripts/StaminaBar.cs

```diff
@@ -42,12 +46,33 @@ void Start()
 
     private void FixedUpdate()
     {
+        if (overrideValue)
+        {
+            UpdateFillAmount(overrideTo);
+            return;
+        }
+
         if (Time.time > lastIncreaseTime + waitUntilDecreaseTime)
         {
             StartCoroutine(DecreaseFillBy(decreasePerSecond * Time.fixedDeltaTime));
         }
     }
 
+    public void InstantIncreaseAwarenessBy(float amount)
+    {
+        UpdateFillAmount(fillImage.fillAmount + amount);
+    }
+
+    public void SetAwakeness(float amount)
+    {
+        UpdateFillAmount(amount);
+    }
+
+    public void ResetAwakeness()
+    {
+        UpdateFillAmount(0f);
+    }
+

```

Comment:
Can we be consistent and choose one from Awareness/Awakeness rather than interchanging? Hue Hue

Reply:
Also maybe change the script name as well :-)

Reply:
LOOL oh boy very true
gonna stick with awakeness because all the main files are named that 
(staminabar to be renamed later)

Reply:
(the rename is coming later so that diffs arent all green, since i made some changes now)